docker-compose.yml

```diff
@@ -0,0 +1,68 @@
+version: '3'
+
+services:
+  node0:
+    container_name: node0
+    image: "tendermint/gaiadnode"
+    ports:
+      - "46656-46657:46656-46657"
+    environment:
+      - ID=0
+      - LOG=$${LOG:-gaiad.log}
+    volumes:
+      - ./build:/gaiad:Z
+    networks:
+      localnet:
+        ipv4_address: 192.168.10.2
```

Comment:
is this file supposed to be ignored in this PR? has custom IPs

Reply:
The file is integral part of the Makefile changes, it's not supposed to be ignored. This file generates a local testnet with 4 nodes on an internal docker network. The custom IPs are static IPs for the nodes, only used within the docker container. There's no impact on your network outside of it.